portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +14,28 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    ArrayList<String> Greetings = new ArrayList();
+    ArrayList<String> commentRepo = new ArrayList<String>();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+      
   }
+  @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOException{
+        String jsonstring = req.getParameter("comment");
+        commentRepo.add(jsonstring);
+        System.out.println(commentRepo);
```

Comment:
Is this logging or debugging? If latter, please remove. If former, I believe there are logger libraries that you should use to log with.